src/app/effects/book.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { EffectsTestingModule, EffectsRunner } from '@ngrx/effects/testing';
+import {TestBed, fakeAsync, tick} from '@angular/core/testing';
+import {BookEffects} from './book';
+import {GoogleBooksService} from '../services/google-books';
+import {Observable} from 'rxjs';
+import {SearchAction, SearchCompleteAction} from '../actions/book';
+import {Book} from '../models/book';
+
```

Comment:
The Atom issue can be solved by inserting `jasmine` into the `types` array of `tsconfig.json`. The type of course needs to be downloaded as well: `npm i @types/jasmine`